backend/contributions/views.py

```
@@ -1100,6 +1100,13 @@ class StewardSubmissionFilterSet(FilterSet):
     has_appeal = BooleanFilter(field_name='has_appeal')
     resubmitted_more_info = BooleanFilter(method='filter_resubmitted_more_info')
 
+    def _split_filter_values(self, value):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_split_filter_values`

(ANN202)